src/app/facility-form/modifier-form/modifier-form.component.ts

```diff
@@ -40,7 +43,7 @@ export class ModifierFormComponent implements OnInit, OnDestroy {
           this.modifierList = res;
         }
       });
-
+    this.parkName = this.facility.pk.substring(this.facility.pk.indexOf('::') + 2);
```

Comment:
There is a second repeated line like this on 72 that could be converted to referencing this variable.

Reply:
[6407bbb](https://github.com/bcgov/parks-reso-admin/pull/121/commits/6407bbba7ac6098a9b395c1b045646d109716085)